### DISCUSSION TITLE
keboard shorts can`t delete and modifed

### DISCUSSION THREAD

**Discussion by WhitCrow** [coder/code-server#2070] [Q&A] [answered]
system：code-server run in  centos7
version: 3.5.0

**Comment by code-asher** [comment, marked as answer]
I tried modifying keyboard shortcuts on our 3.5.0 Docker image but it's working for me. Are you seeing any errors in the logs? (See https://github.com/cdr/code-server/blob/v3.5.0/doc/FAQ.md#how-do-i-debug-issues-with-code-server) Is it with all keyboard shortcuts or specific ones?

**Reply by WhitCrow** [reply]
my fault,dont describe clear.just ''F10'' cant be modified after read it and try again.

**Reply by code-asher** [reply]
As a test I rebound `Show All Commands` to `F10` and it worked (I tried both Firefox and Chrome).

I also tried rebinding `Focus Application Menu` (which defaults to `F10`) to `ctrl+shift+f` and that works. 

I did notice though that this just creates a new entry for `Focus Application Menu` and the default `F10` keybinding is still there. Is this what you're talking about? It doesn't seem like this affects the custom keybindings though; it just means that `F10` will fall back to this.

**Reply by WhitCrow** [reply]
Thank for your reply.  `F10`  was defined to `Focus Application Menu`   as you tried and cant be deleted. it's dirtrubed me that i cant debug `python` when i want define  `F10`  as `Step Over`

**Reply by code-asher** [reply]
Hmm, I have `F10` bound to `Step Over` and when I debug a Python file it seems to be working even though I can't delete the default `Focus Application Menu` keybinding.

If you bind `F10` to `Step Over` and ignore `Focus Application Menu` does `F10` still work when you try to debug and step over or does it still only focus the menu?

**Reply by code-asher** [reply]
Wait nope I'm wrong it definitely doesn't work. I was using `F11` (step into). :facepalm: I'll open an issue.

**Reply by code-asher** [reply]
https://github.com/cdr/code-server/issues/2079

**Reply by WhitCrow** [reply]
Thank you for your attention
